src/api.ts

```diff
@@ -86,13 +86,21 @@ export const getAvailableReciters = async (): Promise<RecitersResponse> =>
 
 /**
  * Get audio file for a specific reciter and chapter.
+ * additionally you can pass `segment: true` to get the timestamps
+ * for each verse and words
  *
  * @param {number} reciterId
  * @param {number} chapter the id of the chapter
  */
 
-export const getAudioFile = async (reciterId: number, chapter: number): Promise<AudioFile> => {
-  const res = await fetcher<AudioFilesResponse>(makeAudioFilesUrl(reciterId, chapter));
+export const getChapterAudioFile = async (
```

Comment:
getChapterAudioData?

Reply:
+1 , yes. `Data` is actually more correct. Since we're not actually fetching the `File`

If we agree on this naming, we should also update other naming that mention `audioFile`, since they are also referring to the same value. For example `types/AudioFile` -> `types/AudioData`.